### DISCUSSION TITLE
Upload to TestRail failed due to a network error. Please make sure you have a valid network connection then try again.

### DISCUSSION THREAD

**Discussion by Shubham22100** [gurock/trcli#245] [General]
I am using Trcli to upload test results in test rail thrugh automation framework but i am getting below message 
Upload to TestRail failed due to a network error. Please make sure you have a valid network connection then try again.

can anyone assist here as its too urgent .

**Comment by Shubham22100** [comment]
can Anyone assist me here ?

**Reply by jakec-routable** [reply]
"Network error" most likely means that the client couldn't make the HTTP request. So you could be using the wrong host domain name, the machine your using doesn't have network access to your TestRail instance, or the network request is getting blocked. Try to debugging the network request. Can you successfully cURL the `[get_projects](https://support.testrail.com/hc/en-us/articles/7077792415124-Projects#getprojects)` api from the machine you are running `trcli`?
